.gitignore

```diff
@@ -0,0 +1,19 @@
+Dan_test
```

Comment:
Is this a real thing?

Reply:
Ya... Not sure, it was in the original .hgignore though....

Reply:
I grepped around for anything that might make something like this and couldn't find anything. I think you get rid of it.